.github/workflows/deploy.yml

```diff
@@ -500,7 +501,7 @@ jobs:
         run: |
           op read "op://${{ vars.OP_VAULT }}/firebase.json/firebase.json" --force --out-file ./firebase.json
           op read "op://${{ vars.OP_VAULT }}/OldApp_AppStore/OldApp_AppStore.mobileprovision" --force --out-file ./OldApp_AppStore.mobileprovision
-          op read "op://${{ vars.OP_VAULT }}/OldApp_AppStore_Share_Extension/OldApp_AppStore_Share_Extension.mobileprovision" --force --out-file ./OldApp_AppStore_Share_Extension.mobileprovision
+          op read "op://${{ vars.OP_VAULT }}/OldApp_AppStore_Share_Extension.mobileprovision/OldApp_AppStore_Share_Extension.mobileprovision" --force --out-file ./OldApp_AppStore_Share_Extension.mobileprovision
```

Comment:
I think this was right before. Looking in the 1P vault, the NewApp item is called `NewApp_AppStore_Share_Extension.mobileprovision`, while the OldApp item does not have the mobileprovision suffix:

<img width="474" alt="image" src="https://github.com/user-attachments/assets/b7d9cb2f-7e76-4992-84fe-ba0603fc603c" />

<img width="459" alt="image" src="https://github.com/user-attachments/assets/18f1f184-2a33-4e40-89cd-d389ebf0f395" />

Reply:
I renamed the share extension ones to remove the `.mobileprovision` from the title because it's confusing

Reply:
I.e.
<img width="706" alt="Screenshot 2025-02-12 at 11 34 30 AM" src="https://github.com/user-attachments/assets/47d71795-89f5-4973-b00f-2173f565bffb" />